schema.sql

```diff
@@ -22,6 +22,24 @@ CREATE TABLE species (
     name varchar(100)
 );
 
+CREATE TABLE vets (
+    id BIGSERIAL PRIMARY KEY,
```

Comment:
- [ ] please make sure that the id can't be null.

Reply:
Thank you for the comment.
I would like to remind you that this was not stated in the requirement

```
Create a table named vets with the following columns:
id: integer (set it as autoincremented PRIMARY KEY)
name: string
age: integer
date_of_graduation: date
```